ukrainian/take_almost_all/take_almost_all.md

```diff
@@ -0,0 +1 @@
+[Беру з собою мотузку, запальничку, ніж та виделку і стрибаю через стіну!](https://github.com/udacity/create-your-own-adventure/blob/master/ukrainian/break-through-wall/break-through-wall.md)
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.